meetings/2025-04-09.md

```diff
@@ -0,0 +1,120 @@
+* eP: For example with SAI it's a little bit more complex, it's not SAI vs type indexes, there is also quite a lot about authorization there, and for the LWS WG there are VCs and data grants in scope, so there is some overlap there.
+* eP: I'm happy that JW refers to requirements, like the WG, the CG needs to work with scopes and requirements. The conversations often seem to be emotional and sentimental. But we should look at which requirements each proposal satisfies.
+* JW: The data grants, I belive are mentikoned in LWS charter. In ESS consent flow VCs are used as part of access requrest mechanism. That has been put into ACP via `acp:vc`. I would have a conversation with Aaron as well as Wouter who is working on UMA flow for CSS. You could see if you can come to rought consensus. Otherwise this can split into different specs.
+* eP: ...
+* JW: I put some links in the chat about other implementations of Inrupt specific flow
+  * https://amc.inrupt.com/login?returnTo=%2F
+  * https://github.com/openwallet-foundation-labs/solid-data-wallet
+* JW: For 711 specifically it would be good to merge it.
+* MdJ: It seems that it cooled down a bit, we can move forward.
+* MdJ: I think we had some conversation of what it means to call our repos specifications. While we are incubating we should be aware of the rest of the Solid project.
+* MdJ: In the future when we make a spec change, we can add a note worning that WG can change it down the road.
+* JZ: So far there has been a very good relation between Practicioners and CG. When we have outreach, we should either do it with Practicioners or joint meeting. For example catalogue, one part would be community focued and another part like shapes fits more CG. We should work on that jointly instead of discussing outreach issues just here in the CG. For example having an Awesome list would be more of Practicioners topic. We will do our part in keeping CG involved on how we are progressing. I would appreciate if CG keeps Practiciones involved when it comes to outreach and community involvement.
+
+### FedCM 
+
+* Th: FedCM is on its way, but the IdP registration feature ( which could strongly improve Solid UX ) is still not part of core. I would like to share the current situation, and gather advice and feedback ( from people you have experience with the W3C process ) on how we can build a stronger case to make this feature part of FedCM core. 
+* Th: I'm not used to W3C. Idp registration feature, which is needed for Solid is not part of FedCM and still working on. I don't know how to proceed with google developers which are leading the spec.I don't want to implement it if there is no usage. I don't know if it is only Google's thing to be in that position to listen of what peole want. This feature doesn't have direct interest from google since they have strong position in IdP world. I'm bit lost in all that. 
```

Comment:
```suggestion
* Th: I'm not used to W3C. IdP registration feature, which is needed for Solid, is not part of FedCM and still being worked on. I don't know how to proceed with Google developers who are leading the spec. I don't want to implement it if there is no usage. I don't know if it is only Google's thing to be in that position to listen to what people want. This feature doesn't have direct interest from Google since they have a strong position in IdP world. I'm a bit lost in all that. 
```